packages/components/src/Avatar/_docs/Avatar.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import { Meta, StoryObj } from "@storybook/react"
+import { classNameOverrideArgType } from "~storybook/argTypes"
+import { Avatar } from "../index"
+
+const meta = {
+  title: "KAIO-staging/Avatar",
+  component: Avatar,
+  argTypes: {
+    ...classNameOverrideArgType,
+    isCompany: { type: "boolean" },
+    isCurrentUser: { type: "boolean" },
+    disableInitials: { type: "boolean" },
+    fullName: { type: "string" },
+    avatarSrc: { type: "string" },
```

Comment:
I found a fix for this :)
The `shouldRemoveUndefinedFromOptional` flag removes the `undefined` which makes the control an object.

https://github.com/storybookjs/storybook/issues/14523#issuecomment-817823037